bls12-381/src/main/java/org/hyperledger/besu/nativelib/bls12_381/LibEthPairings.java

```diff
@@ -55,4 +56,21 @@ public static native int eip2537_perform_operation(
       IntByReference o_len,
       byte[] err,
       IntByReference err_len);
+
+  public static final int EIP196_PREALLOCATE_FOR_ERROR_BYTES = 256;
+
+  public static final int EIP196_PREALLOCATE_FOR_RESULT_BYTES = 64;
+
+  public static final byte EIP196_ADD_OPERATION_RAW_VALUE = 1;
+  public static final byte EIP196_MUL_OPERATION_RAW_VALUE = 2;
+  public static final byte EIP196_PAIR_OPERATION_RAW_VALUE = 3;
+
+  public static native int eip196_perform_operation(
```

Comment:
i would say that the names of the parameters are not very explicit. i guess you took those from the native implementation ?

Reply:
Yes, this name must match the library.